app/server.py

```diff
@@ -181,6 +196,54 @@ def index():
     return "Hello, %s!" % g.user
 
 
+@app.route("/channels")
+@multi_auth.login_required
+def get_channels():
+    
+    channels = database.get_channels(g.user)
+    
+    for channel in channels:
+        channel['users'] = database.get_users_for_channel(channel['id'])
```

Comment:
I will fix this query

Reply:
This implementation takes 0.0060±0.0030 seconds (95% CI) for a user with 8 channels. I will investigate how an alternative query will perform.

Reply:
Is it ok if this stays as is, and in a later version the for-loop should be discarded altogether and replaced by client-side caching and on-demand fetching.